packs/feats/magic-warrior-dedication.json

```
@@ -32,7 +32,9 @@
             {
                 "key": "FlatModifier",
                 "label": "vs. divination (Magic Warrior Dedication)",
-                "predicate": [],
+                "predicate": [
+                    "against-divination"
+                ],
```

```suggestion
                "predicate": [
                    "divination"
                ],
```
This will let it work automatically for legacy items.